src/client_side.cc

```diff
@@ -657,28 +656,26 @@ clientSetKeepaliveFlag(ClientHttpRequest * http)
 }
 
 /// checks body length of non-chunked requests
```

Comment:
> PR description: Versions 0.9 and 1.0 without a Content-Length are forbidden a body due to cache
corruption possibilities.

The function documentation and usage implies that this function does not apply to chunked requests. The PR description can be reasonably interpreted as if the changes do apply to chunked requests. Please resolve this conflict one way or another.

Reply:
I disagree that is a reasonable interpretation of a line which specifically talks about versions where chunked is not valid.

Reply:
> I disagree that is a reasonable interpretation of a line which specifically talks about versions where chunked is not valid.

I think you are implying that HTTP/1.0 messages with chunked encoding are invalid. IIRC, Squid considers HTTP/1.0 messages with chunked encoding _valid_, and I think Squid (including related documentation) should continue to do so.

Reply:
description updated earlier